tests/OpeningHoursScenarioTest.php

```diff
@@ -21,7 +21,7 @@ public function setup()
      **/
     public function testOpenNowIsOpen()
     {
-        $this->doRequest('GET', '/api/services/1/open-now?testDateTime=2017-09-05 10:00:00');
+        $this->doRequest('GET', '/api/v1/services/1/open-now?testDateTime=2017-09-05 10:00:00');
```

Comment:
perhaps refactor test code by settign a protected $apiUrl = '/api/v1'; like the other tests
in case of version change, it won't have to be changed in as many places as now done